owncloudData/src/main/java/com/owncloud/android/data/user/db/UserDao.kt

```diff
@@ -33,6 +35,11 @@ interface UserDao {
         accountName: String
     ): UserQuotaEntity?
 
+    @Query(SELECT_QUOTA)
+    fun getQuotaForAccountAsFlow(
+        accountName: String
+    ): Flow<UserQuotaEntity>
```

Comment:
Why with the same SQL query you have a nullable result in the previous method but not here?

Reply:
Added nullability to keep the same structure in both SQL queries